catalogue_graph/pyproject.toml

```diff
@@ -19,7 +21,14 @@ dependencies = [
     "python-dateutil",
     "types-python-dateutil",
     "pyarrow~=20.0",
-    "pyarrow-stubs"
+    "pyarrow-stubs",
+    "lxml>=6.0.0",
+    "mypy>=1.16.1",
```

Comment:
This is already in the dev dependency group. I don't think it's needed at runtime?